components/ai_chat/BUILD.gn

```diff
@@ -54,3 +55,20 @@ mojom("mojom") {
   sources = [ "ai_chat.mojom" ]
   public_deps = [ "//mojo/public/mojom/base" ]
 }
+
+source_set("ai_chat_api_unit_tests") {
```

Comment:
nit: just call it `unit_tests`?

Reply:
This test has been removed. All test should be inside APIHelper.